sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -404,6 +419,77 @@ def save(self, results):
     return self._client.insert_rows(self._bq_table, results)
 
 
+class InfluxDBMetricsPublisherOptions(object):
+  def __init__(
+      self,
+      measurement,  # type: str
+      db_name,  # type: str
+      hostname='http://localhost:8086',  # type: str
```

Comment:
Why do we need this default value here? Isn't it provided from pipeline options default value?

Reply:
There are some minor chances that someone would use InfluxDBPublisher in their code without casting pipeline options to LoadTestOptions (view_as(...)). But beside that, there's no particular reason

Reply:
Then maybe we should have param without default value? I can imagine case when someone doesn't know why metrics tries to be saved to this host without checking this code deeper?

Reply:
Ok then. I will remove the default value.